src/go/k8s/api/vectorized/v1alpha1/cluster_webhook.go

```diff
@@ -284,6 +286,25 @@ func (r *Cluster) validateScaling() field.ErrorList {
 	return allErrs
 }
 
+func (r *Cluster) validatePriorityClassName() field.ErrorList {
```

Comment:
IMO there's no need to validate this. This error will eventually be encountered when pods are trying to be scheduled. It's more of the "kubernetes way" to let misconfigurations fail in a backoff loop rather than validate them up front; for better or worse.

I don't have super strong feelings here if the existing validators do something similar.

Reply:
Understood. I agree. Just follow the existing pattern. I will keep it.